addons/hr_timesheet/views/project_sharing_views.xml

```diff
@@ -22,12 +22,12 @@
                 <page string="Timesheets" id="timesheets_tab" attrs="{'invisible': [('allow_timesheets', '=', False)]}">
                     <group>
                         <group>
-                            <div class="o_td_label">
-                                <label for="planned_hours" string="Allocated Hours" class="me-2" attrs="{'invisible': [('encode_uom_in_days', '=', True)]}"/>
-                                <label for="planned_hours" string="Allocated Days" class="me-2" attrs="{'invisible': [('encode_uom_in_days', '=', False)]}"/>
-                                <field name="planned_hours" class="oe_inline" widget="timesheet_uom_no_toggle"/>
+                            <label for="planned_hours" string="Allocated Hours" attrs="{'invisible': [('encode_uom_in_days', '=', True)]}"/>
+                            <label for="planned_hours" string="Allocated Days" attrs="{'invisible': [('encode_uom_in_days', '=', False)]}"/>
+                            <div class="o_row">
+                                <field name="planned_hours" class="o_field_float_time oe_inline ms-2" widget="timesheet_uom_no_toggle"/>
```

Comment:
This diff can be done in master but not in master, I suggest to put your last changes instead and in master, you will do that. 🙂 

Because here you changes the xml structure and so you could break the customer customisation (xpath).

Reply:
yes, that's correct. my bad :see_no_evil: